lib/iris/fileformats/pp.py

```diff
@@ -673,8 +673,8 @@ def __init__(self, value):
 
         """
         if value < 0:
-            raise ValueError('Negative numbers not supported with '
-                             'splittable integers object')
+            msg = 'Setting LBPROC with a negative value of {}'
```

Comment:
This warning, whilst correct, might be interpreted as meaning LBPROC is stored somewhere in the resulting cube, which may lead to confusion rather than clarification. How about: ```Negative values of LBPROC ({}) are not supported.```? I think this makes it clearer that we don't handle this case.